thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -210,22 +210,24 @@ def forward(
         """Forward method for adaptive evaluation
 
         Args:
-            ref_feature: feature from reference view
-            src_features: features from source views
-            ref_proj: projection matrix of reference view
-            src_projs: source matrices of source views
-            depth_sample: sample depth map,
-            depth_min: minimum virtual depth,
-            depth_max: maximum virtual depth,
+            ref_feature: feature from reference view, (B, C, H, W)
+            src_features: features from (Nview-1) source views, (Nview-1) * (B, C, H, W)

```

Comment:
where is Nview defined?

Reply:
`Nview` is the number of input images for PatchmatchNet to test (one reference view and `(Nview-1)` source view). I thought it matched with `Ndepth` notation in the same style used in the original comments